lib/plain/common-types.ts

```diff
@@ -388,15 +392,33 @@ export type PlainClientAPI = {
     ): Promise<BulkActionProps<BulkActionValidatePayload>>
   }
   comment: {
+    get(
+      params: OptionalDefaults<GetCommentParams>,
+      headers: CommentBodyFormatHeader
+    ): Promise<RichTextCommentProps>
     get(params: OptionalDefaults<GetCommentParams>): Promise<CommentProps>
+    getMany(
+      params: OptionalDefaults<GetManyCommentsParams & QueryParams>,
+      headers: CommentBodyFormatHeader
+    ): Promise<CollectionProp<RichTextCommentProps>>
     getMany(
       params: OptionalDefaults<GetManyCommentsParams & QueryParams>
     ): Promise<CollectionProp<CommentProps>>
+    create(
+      params: OptionalDefaults<CreateCommentParams>,
+      rawData: RichTextBodyParam,
+      headers?: AxiosRequestHeaders
```

Comment:
Not sure how it works, I would imagine that `AxiosRequestHeaders` is a generic enough and does not enforce the `contentful-comment-body-format` to be `rich-text` so we don't guarantee the response type in this case.

Reply:
The response type is guaranteed by the `rawData`  that can be either a string or a `Rich Text Document`